CHANGELOG.md

```diff
@@ -36,6 +38,8 @@ Thank you to everyone else that has contributed by reporting bugs, enhancements
 |               | `--freyja_lineages`       |
 |               | `--skip_freyja_boot`      |
 |               | `--additional_annotation` |
+|               | `--min_contig_length`     |
+|               | `--min_perc_cgaligned`    |
```

Comment:
I'm not certain this variable is very descriptive. What about `--min_perc_contig_aligned` or `--blast_perc_aligned` or something else?

Reply:
I named it this way because the column is `%cgAligned`. But I can name it other way.

Reply:
In my opinion, the %cgAligned will call for questions if you see this for the first time.

Reply:
mmmm I think I'll go with `--min_perc_contig_aligned` and write a more exhaustive description of the BLAST headers in the documentation. The `%cgAligned` column has always been there, but I think it is not properly described. Thanks!